Backend-Roadmap/.gitignore

```diff
@@ -0,0 +1,2 @@
+.vs/*
```

Comment:
Check the gitignore to include all temporary files

Reply:
.ignore actualizado